### ISSUE TITLE
Pull images from links

### ISSUE THREAD

**Issue by TmsLsk** [COVID-science-engine/crowdsourced-papers-prototype#9]
If readily achieved, can we pull images from the links and have them displayed in the preview such as Twitter does?

![sienceenginepic](https://user-images.githubusercontent.com/63466798/79049869-0b612380-7c1e-11ea-9cd8-80abc6504d56.JPG)

![twitterpic](https://user-images.githubusercontent.com/63466798/79049875-10be6e00-7c1e-11ea-96af-9d9b4b10a674.JPG)

**Comment by MatMoore** [comment]
Yep this is doable. It will depend on the website was image is available but we should be able to do something equivalent to what twitter does.

@SIMC4774 do you have any thoughts on the design of this?

I haven't looked into the implementation yet but it looks like there are [a few libraries available for this](https://yuktikhurana.github.io/Slack-Like-Url-Unfurling-In-Rails/) that we could use.
